examples/lighting-app/esp32/main/main.cpp

```diff
@@ -129,6 +134,10 @@ static void InitServer(intptr_t context)
 
     static Tracing::Insights::ESP32Backend backend;
     Tracing::Register(backend);
+// Setting the sampling time to 60 seconds i.e 60,000 ms
+#if CONFIG_ENABLE_ESP_INSIGHTS_SYSTEM_STATS
+    chip::System::Stats::InsightsSystemMetrics::GetInstance().RegisterAndEnable(chip::System::Clock::Timeout(60000));
```

Comment:
Nit: Can use a macro for this 60000 with the right name, so that the comment can also be removed